### PR TITLE
Include Kazakh version

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
PR created with the documents brought by Ilyas Arystanbekov (2014ARYS01) and Sultanbibarys Omar (2017SULT02).

### CODE REVIEW

File: kazakh/wca-regulations.md [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+
+## Ескертулер
+
+### Аударма
+
+Бұл ресми WCA ережелері емес, бірақ тегін аударма. Күдік туындаған жағдайда бастапқы Ережеге жүгініңіз. Аудармада қателер болуы мүмкін! Аударуға қатысты сұрақтар мен ескертулерді [iarystanbekov@worldcubeassociation.org](mailto:iarystanbekov\@worldcubeassociation.org)электрондық поштасына жіберіңіз.
+
+Кейбір терминдер әдейі аударылмаған, өйткені қазақ тілінде сәйкес сөздер жоқ, немесе тіпті орыс жарыстарында ағылшын атаулары қолданылады.Оларға мыналар жатады: скрамблер (жұмбақтарды араластыратын адам), square-1, Clock (жұмбақтардың аттары) және т.б.
+
+Аударманы: [Ильяс Арыстанбеков](https://www.worldcubeassociation.org/persons/2014ARYS01), Сұлтанбибарыс Омар (https://www.worldcubeassociation.org/persons/2017SULT02) жүргізді.
```
Comment (Ivan-Brigidano) [inline kazakh/wca-regulations.md:14]: I don't think the credit section should be included on the Regulations file.

Comment (lgarron) [reply kazakh/wca-regulations.md:14]: > I don't think the credit section should be included on the Regulations file.

Given that it doesn't fit on the translations page anymore, I think on in the document itself is best. We could standardize the section, but I think allowing it directly inline for now is sufficiantly good.

File: kazakh/wca-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
+# <wca-title> WCA косымша
+
+<version>Нұсқасы: 1 қаңтар 2020 ж.
+
+Жылғы [official: 0d1d383]
```
Comment (viroulep) [inline kazakh/wca-guidelines.md:5]: Please don't include that, it's generated automatically.

File: kazakh/wca-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
+# <wca-title> WCA косымша
+
+<version>Нұсқасы: 1 қаңтар 2020 ж.
+
+Жылғы [official: 0d1d383]
+
+## Ескертулер
+
+### Wca Ережелерi
```
Comment (viroulep) [inline kazakh/wca-guidelines.md:9]: This should probably be "WCA".

File: kazakh/wca-regulations.md (hunk 1 above)
Comment (Ivan-Brigidano) [reply kazakh/wca-regulations.md:14]: That makes sense.

**Pull request merged**